src/postgraphql/schema/createPostGraphQLSchema.ts

```diff
@@ -14,6 +15,18 @@ import getPgProcedureComputedClass from './procedures/getPgProcedureComputedClas
 import getPgTokenTypeFromIdentifier from './auth/getPgTokenTypeFromIdentifier'
 import getJwtGqlType from './auth/getJwtGqlType'
 
+async function writeFileAsync (
+  path: string,
+  contents: string,
+): Promise<void> {
+  await new Promise((resolve, reject) => {
+    writeFile(path, contents, error => {
+      if (error) reject(error)
+      else resolve()
+    })
+  })
+}
+
```

Comment:
I duplicated this function from elsewhere in the code, not sure the best way to go about this.